### PR TITLE
MOSIP-29666 Update identity-mapping.json

### PR DESCRIPTION
**Pull request by kameshsr**
Updated identity-mapping.json according to print policy

### CODE REVIEW

File: identity-mapping.json [hunk 1]
```diff
@@ -69,14 +69,59 @@
 		"locationHierarchyForProfiling": {
 			"value": "zone,postalCode"
 		},
-		"addressLine1": {
-			"value": "addressLine1"
+		"presentAddressLine1": {
```
Comment (vishwa-vyom) [inline identity-mapping.json]: normally in identity mapping json file, we can change only the value, the keys as it is will be used in the code base

File: identity-mapping.json [hunk 2]
```diff
@@ -69,14 +69,59 @@
...
+			"value": "presentProvince"
+		},
+		"presentCity": {
+			"value": "presentCity"
+		},
+		"presentBarangay": {
+			"value": "presentBarangay"
+		},
+		"presentZipcode": {
+			"value": "presentZipcode"
```
Comment (vishwa-vyom) [inline identity-mapping.json]: Do we need any all these new values ? Which code is using all these ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: identity-mapping.json (hunk 2 above)
Comment (kameshsr) [reply identity-mapping.json]: The automation team is using these values while activating UIN in /identity API.

File: identity-mapping.json (hunk 1 above)
Comment (kameshsr) [reply identity-mapping.json]: Yes sir corrected.

File: identity-mapping.json (hunk 2 above)
Comment (vishwa-vyom) [reply identity-mapping.json]: If we are adding this only for automation tools compatibility, I would like to understnad this from the automation team better.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**